app/styles/ui/window/_title-bar.scss

```diff
@@ -49,6 +49,7 @@
   // automatically even for borderless window so we only render
   // controls on Windows.
   .window-controls {
+    order: 3;
```

Comment:
This seems awfully fragile and easy for us to regress on. Could we at least add a comment about why this is important?